pyne/data.pyx

```diff
@@ -1044,6 +1044,52 @@ def decay_children(nuc, use_metastable=True):
 
     return dc
 
+def all_children(nuc):
+    """
+    returns child nuclides from both level and decay data
+
+    Parameters
+    ----------
+    nuc : int
+        input nuclide in state id form
+
+    Returns
+    -------
+    ids : list of ints
+        list of all known decay children from ensdf data.
+    """
+
+    return list(set(decay_children(nuc, False)) & set(decay_data_children(nuc)))
```

Comment:
Also, shouldn't this be `set(decay_children(nuc, False)) | set(decay_data_children(nuc))`?  That is, don't we want the union and not the intersection?